apps/rush/src/start.ts

```diff
@@ -1,39 +1,19 @@
 // Copyright (c) Microsoft Corporation. All rights reserved. Licensed under the MIT license.
 // See LICENSE in the project root for license information.
 
-// The minimal set of imports that are safe even for ancient NodeJS versions:
+// The minimal set of imports that are safe even for ancient Node.js versions:
 import * as colors from 'colors';
 import * as os from 'os';
 import * as semver from 'semver';
 
-const nodeVersion: string = process.versions.node;
+import { NodeJsCompatibility } from '@microsoft/rush-lib/lib/logic/NodeJsCompatibility';
```

Comment:
This should really be marked as `@internal` rather than accessed via a path-based import.  That's what `@internal` is for.  We should ideally work to eliminate all the path-based imports from our code base.

Reply:
I used a path-based import in this one case because this code runs before we eval the other (more complex) imports.

The reasoning is that we should provide a meaningful error message before the tool crashes if someone is on a really old NodeJS version that, for whatever reason, fails to evaluate the imports.

Reply:
That's persuasive.  Maybe add a comment explaining it.